nautobot/utilities/templates/utilities/templatetags/filter_form.html

```diff
@@ -0,0 +1,52 @@
+{% load form_helpers %}
```

Comment:
Would it make sense to name this file to match the templatetag, i.e. `dynamic_filter_form_modal.html` instead?